packages/core/contracts/oracle/implementation/Staker.sol

```diff
@@ -113,14 +114,33 @@ abstract contract Staker is StakerInterface, Ownable, Lockable {
      * be added to the pending stake. If not, the stake amount will be added to the active stake.
      * @param amount the amount of tokens to stake.
      */
-    function stake(uint256 amount) public override nonReentrant() {
-        VoterStake storage voterStake = voterStakes[msg.sender];
+    function stake(uint256 amount) public {
+        _stakeTo(msg.sender, msg.sender, amount);
+    }
+
+    /**
+     * @notice Pulls tokens from sender wallet and stakes them for the recipient. If we are in an active reveal phase the
+     * stake amount will be added to the pending stake. If not, the stake amount will be added to the active stake.
+     * @param recipient the recipient address.
+     * @param amount the amount of tokens to stake.
+     */
+    function stakeTo(address recipient, uint256 amount) public {
+        _stakeTo(msg.sender, recipient, amount);
+    }
```

Comment:
Just want to call this out as a new feature.

_Anyone_ can add money to your account. We might want to just briefly run this by Product.